config/executor/config.yaml

```diff
@@ -47,8 +47,8 @@ kubernetes:
   stuckTerminatingPodExpiry: 1m
   podKillTimeout: 5m
   minimumResourcesMarkedAllocatedToNonArmadaPodsPerNode:
-    cpu: 100m
-    memory: 50Mi
+    cpu: 1
```

Comment:
I was seeing preemptions locally due to overcommitting the kind node without this.